e2e/evaluations_defaut_strategy.ts

```diff
@@ -37,11 +38,12 @@ test('boolVariation', async (t) => {
 test('stringVariation', async (t) => {
   const { bktClient, defaultUser } = t.context;
   t.is(await bktClient.stringVariation(defaultUser, FEATURE_ID_STRING, ''), 'value-1');
-  t.deepEqual(
+  assetEvaluationDetails(
+    t,
     await bktClient.stringVariationDetails(defaultUser, FEATURE_ID_STRING, ''),
     {
       featureId: FEATURE_ID_STRING,
-      featureVersion: 4,
+      featureVersion: 22,
```

Comment:
Let's remove the feature version from the checking to avoid future issues when editing the flags on the admin console. Same for other tests.

Reply:
the assert logic is [here](https://github.com/bucketeer-io/node-server-sdk/pull/85/files#diff-152e3620e12a0becd608ff997ee0dfb6136a573c2732491b21cfdaf1e8f98272R5) , the line that you are comment is a BKTEvaluationDetails object.